src/btree/bt_vrfy.c

```diff
@@ -355,6 +367,11 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
             WT_ERR(__dump_layout(session, vs));
     }
 
+    /* Report WT_NOTFOUND if a checkpoint to verify was specified but not found. */
```

Comment:
In `api_data.py`, you mention `ENOENT`, does `WT_NOTFOUND` get translated into `ENOENT`?

Reply:
Oh yes, [here](https://github.com/wiredtiger/wiredtiger/pull/11919/files#diff-12a8d551424bbca1375b3fb40adac258f7addd9cc14281393318477e6caca9feR221). My bad!